deploy-manage/tools/snapshot-and-restore/s3-repository.md

```diff
@@ -87,10 +90,7 @@ The following list contains the available client settings. Those that must be st
 :   An S3 session token. If set, the `access_key` and `secret_key` settings must also be specified.
 
 `endpoint`
-:   The S3 service endpoint to connect to. This defaults to `s3.amazonaws.com` but the [AWS documentation](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region) lists alternative S3 endpoints. If you are using an [S3-compatible service](#repository-s3-compatible-services) then you should set this to the service’s endpoint.
-
-`protocol`
```

Comment:
This setting is now in use again, see #127744. It's deprecated, users should be setting `endpoint` to a full URL including scheme, but if they omit the scheme then for now we get it from this setting.

I support moving the info about the trust store to the `endpoint` setting tho.